saos-restful-web-api/src/main/java/pl/edu/icm/saos/api/search/judgments/JudgmentsListSuccessRepresentationBuilder.java

```diff
@@ -53,6 +88,9 @@
 
     @Autowired
     private SearchJudgmentItemMapper judgmentItemMapper;
+    
+    @Autowired
+    private ParametersExtractor parametersExtractor;
```

Comment:
I think it would be better to have:
   @Value("${restful.api.max.page.size}")
   private int maxPageSize=100;
    
    @Value("${restful.api.min.page.size}")
    private int minPageSize=10;

instead of ParametersExtractor